numpy/random/mtrand/mtrand.pyx

```diff
@@ -4203,9 +4203,9 @@ cdef class RandomState:
         Draw samples from a Hypergeometric distribution.
 
         Samples are drawn from a hypergeometric distribution with specified
-        parameters, ngood (ways to make a good selection), nbad (ways to make
-        a bad selection), and nsample = number of items sampled, which is less
-        than or equal to the sum ngood + nbad.
+        parameters, ``ngood`` (ways to make a good selection), ``nbad`` (ways to make
+        a bad selection), and ``nsample`` = number of items sampled, which is less
```

Comment:
nit: replace `= description` with `(description)` to match the other two descriptions here